JiT update....

leroy-douglas opened the pull request.

leroy-douglas committed:
JiT update....

Rody-Kirwan commented on week2/2-website/index.html:
```diff
@@ -16,6 +16,63 @@
+                <button>Learrn More</button>
+             </div> 
+         </div>
+        <main id=benefits>
```
```suggestion
        <main id="benefits">
```
missing double quotations :)

Rody-Kirwan commented on week2/2-website/index.html:
```diff
@@ -16,6 +16,63 @@
+            </div>
+        </main>
+        <footer>
+            <h4></h4>
```
remove the empty `h4`

Rody-Kirwan commented on week2/2-website/css/style.css:
```diff
@@ -16,4 +24,230 @@ body {
+@media (min-width: 76.8em) {
+    .content-container {
+        width: 73.6em;
+        sbackground-color: red;
```
seems to be a typo here
```suggestion
        background-color: red;
```

GTabala replied:
I think Leroy uses this instead of commenting.

Rody-Kirwan commented on week2/2-website/css/style.css:
```diff
@@ -16,4 +24,230 @@ body {
+@media (min-width: 99.2em) {
+    .content-container {
+        width: 96rem;
+        sbackground-color: blue;
```
same

Rody-Kirwan commented on week2/2-website/css/style.css:
```diff
@@ -16,4 +24,230 @@ body {
+@media (min-width: 120rem) {
+    .content-container {
+        width: 60%;
+        sbackground-color: green;
```
same

Rody-Kirwan commented on week2/2-website/css/style.css:
```diff
@@ -16,4 +24,230 @@ body {
+    color: #3f729b;
+}
+
+.fa{
```
better to create your own classname rather than setting styles directly on fontawsome class.
Consider that later you might want to have a bigger icon which will mean you'll have to overwrite this. and then reset it again for smaller icons. Maybe consider creating classes like
`icon-sm` `icon-md` and `icon-lg`.
Then you can do something like
`<i class="fa fa-twitter icon-sm">`

Also I'm not sure you need to set the font-family

Rody-Kirwan commented on week2/2-website/css/style.css:
```diff
@@ -16,4 +24,230 @@ body {
+}
+
+nav.menu  {
+   /*  float: right; */
```
remove any unnecessary / commented code before submitting a PR.

Rody-Kirwan reviewed (approved):
Nice work Leroy.

left a couple of small notes but otherwise really good.

leroy-douglas committed:
Tidied up html & css

Removed unecessary code and comments.
Also used the proper class to size the social media icons.
